Sources/AppHost/OsmAndAppImpl.mm

```diff
@@ -377,6 +378,11 @@ - (BOOL) initialize
             OAAppSettings.sharedManager.availableApplicationModes = @"car,bicycle,pedestrian,public_transport,";
         }
         [[NSUserDefaults standardUserDefaults] setFloat:currentVersion forKey:@"appVersion"];
+        [OAAppSettings sharedManager].shouldShowWhatsNewScreen = YES;
+    }
+    else
+    {
+        [OAAppSettings sharedManager].shouldShowWhatsNewScreen = NO;
```

Comment:
Do not set this in app initialization! Set it to NO after you've shown the what's new dialogue